MedscribeUI/src/stories/selector.stories.tsx

```diff
@@ -0,0 +1,21 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import NoteControlsLayout from '../components/NoteControls/NotecontrolsLayout';
+import { MantineProvider } from '@mantine/core';
+import '@mantine/core/styles.css';
+
+const Wrapper = () => {
+  return (
+    <MantineProvider>
+      <NoteControlsLayout />
```

Comment:
can you please wrap this component because we want to make sure we can play with the sizing to ensure that this component scales nicely with different layouts. Try putting these in as arguments or just create multiple exported components ex: export const WrappedBig: Story = {};

Reply:
I created a wrapper so its mroe modular but let me know if thats not what you meant. I am a little confused by this